mathics/builtin/strings.py

```diff
@@ -1688,6 +1695,31 @@ def apply_form(self, value, form, evaluation, options):
         return String(text)
 
 
+class InterpretedBox(PrefixOperator):
+    """
+    <dl>
+      <dt>'InterpretedBox[$box$]'
+      <dd>is the ad hoc fullform for \! $box$. just
+          for internal use...
+
+    >> \! \(2+2\)
+     = 4
+    </dl>
+    """
+
+    operator = "\\!"
+    precedence = 670
```

Comment:
@mmatera Just curious - how did you arrive at the value 670?

Reply:
This seems to be the default in WL...
```
In[1]:= Precedence[anycustomsymbol]
Out[1]= 670
```

Reply:
Thanks - for the information.

Reply:
Just a comment:  `InterpretedBox` is not a symbol in WL. However, someone else introduced it in the MathicsScanner (`\!`,  `\(`, etc) were already implemented in the scanner. I just hooked it to a new class that provides the basic elements needed for the parser, and a more or less simple way to reproduce the basic behavior.

Reply:
Ok. As with too many other things, I guess we will address in a more complete fashion latter. 

For now this doesn't seem too disruptive or complex.